soc_curve.py

```diff
@@ -46,6 +46,8 @@ def voltage_to_soc(self, cell_voltage: float) -> float:
 
     @staticmethod
     def soc_to_voltage(soc: float):
+        assert 0.0 <= soc <= 1.0

```

Comment:
works out of range too, assert not needed

Reply:
It guards against a misunderstanding of soc_to_voltage(15) where someone might think its 15%.